assets/src/stories-editor/components/higher-order/with-amp-story-settings.js

```diff
@@ -300,6 +317,34 @@ export default createHigherOrderComponent(
 							</RotatableBox>
 						</ResizableBox>
 					) }
+					{ ! needsResizing && (
+						<RotatableBox
+							blockElementId={ `block-${ clientId }` }
+							initialAngle={ rotationAngle }
+							className="amp-story-editor__rotate-container"
+							angle={ rotationAngle }
```

Comment:
Looks like this causes prop type invalidation when `rotationAngle` is undefined. I guess we need to make the prop optional in the prop types definition.

Will have a look.